specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json

```diff
@@ -0,0 +1,2367 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "v3.1-preview.4",
+    "contact": {
+      "name": "Microsoft Cognitive Services",
+      "url": "https://azure.microsoft.com/en-us/services/cognitive-services/text-analytics/",
+      "email": "mlapi@microsoft.com"
+    },
+    "title": "Text Analytics Client",
+    "description": "The Text Analytics API is a suite of natural language processing (NLP)  services built with best-in-class Microsoft machine learning algorithms.  The API can be used to analyze unstructured text for tasks such as sentiment analysis,  key phrase extraction and language detection. Further documentation can be found in <a href=\"https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/overview\">https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/overview</a>"
```

Comment:
We should include the description for Analyze and Healthcare.

Reply:
I'll add a mention of healthcare in my PR, but don't think the general description needs to mention analyze as its a wrapper around other tasks a user would want to access.